client/state/stored-cards/test/selectors.js

```diff
@@ -0,0 +1,36 @@
+// External dependencies
+import { expect } from 'chai';
+
+// Internal dependencies
+import { getCards, getByCardId } from '../selectors';
+import { STORED_CARDS } from './fixture';
+
+describe( 'selectors', () => {
+	describe( 'getCards', () => {
+		it( 'should return a purchase by its ID, preserving the top-level flags', () => {
+			const state = {
```

Comment:
I almost forgot, we are wrapping state with `deepFreeze` to make sure it doesn't get mutated.